client/src/partials/journal/modals/journalErrorViewer.modal.js

```diff
@@ -0,0 +1,46 @@
+angular.module('bhima.controllers')
+  .controller('JournalErrorViewerModalController', JournalErrorViewerModalController);
+
+JournalErrorViewerModalController.$inject = [
+  '$uibModalInstance', 'GridColumnService', 'GridGroupingService', 'JournalErrorViewerService', 'params'
+];
+
+/**
+ * @module journal/modals/journalErrorViewer.modal
+ *
+ * @description
+ * This controller provides an error report relative to the posting operation
+ */
+function JournalErrorViewerModalController (ModalInstance, Columns, Grouping, JournalErrorViewerService, Params) {
+  var vm = this;
+  var cacheKey = 'errorReport';
+  var columns = [
+    { field : 'code', displayName : 'TABLE.COLUMNS.ERROR_TYPE', headerCellFilter : 'translate', enableCellEdit : false, allowCellFocus : false},
```

Comment:
I think you can set `enableCellEdit: false` on the gridOptions so that you don't have to specify it on each column.  It might also work that way for `allowCellFocus`.